src/full-monero-node-external-sd-install.sh

```diff
@@ -43,23 +45,104 @@ termux-wake-lock -y
 pkg update -y
 pkg install nano wget termux-api jq -y
 
-# Pre-Clean Old Setup
-rm -f $TERMUX_BOOT/before_start_monero_node
-rm -f $TERMUX_BOOT/*XMR\ Node*
-
 # Create Directories
+
 mkdir -p $MONERO_CLI
-mkdir -p $NODE_DATA
+mkdir -p $NODE_CONFIG
 mkdir -p $TERMUX_BOOT
 mkdir -p $TERMUX_SHORTCUTS
 mkdir -p $TERMUX_SCHEDULED
-mkdir -p $NODE_CONFIG
 
+# Pre-Clean Old Setup
+rm -f $TERMUX_BOOT/before_start_monero_node
+rm -f $TERMUX_BOOT/*XMR\ Node*
+NODE_CONFIGOLD=~/monero-cli/config
```

Comment:
`NODE_CONFIG_OLD`

Reply:
There's a bunch of these (with no underscore separating the words).

Is this just aesthetic?  

ASKSD
INTFREE
MOVEDATA
CONFIRMINT
NODE_CONFIGOLD